lms/templates/dashboard.html

```diff
@@ -154,25 +159,31 @@
         <div class="side-widget side-profile">
           <div class="side-profile-row">
             <div class="avatar">
-              <img src="http://polymath.colaraz.net/cvsocial/_graphics/icons/user/defaultsmall.gif" alt="asif rana" title="asif rana">
+              <img src="http://polymath.colaraz.net/cvsocial/_graphics/icons/user/defaultsmall.gif" alt="${user.profile.name}" title="${user.profile.name}">
             </div>
-            <span class="user-name">asif rana</span>
-            <span class="designation">Student</span>
+            <span class="user-name">${user.profile.name}</span>
+            % if user.is_staff:
+              <span class="designation">Staff</span>
+            % else:
+              <span class="designation">Student</span>
+            % endif
           </div>
           <div id="progress" class="profile-progress-bar">
             <div class="progress progress-striped">
-              <div class="progress-bar progress-bar-striped" style="width:20%;background-color:#eb7d3c;" id="progressbar">
-                <span class="sr-only">Beginner</span>
+              <div class="progress-bar progress-bar-striped" style="width:${level_percentage}%;background-color:#eb7d3c;" id="progressbar">
+                <span class="sr-only">${level_text}</span>
               </div>
             </div>
-            <span class="progress-result">Beginner</span>
+            <span class="progress-result">${level_text}</span>
           </div>
         </div>
         <div class="side-widget side-nav">
           <ul>
-            <li class="active"><a href="#">Courses</a></li>
-            <li><a href="#">Discover New</a></li>
-            <li><a href="#">Studio</a></li>
+            <li class="active"><a href="${reverse('dashboard')}">Courses</a></li>
+            <li><a href="${marketing_link('COURSES')}">Discover New</a></li>
+            % if user.is_staff:
+              <li><a href="//${settings.CMS_BASE}">Studio</a></li>
```

Comment:
Let's create a variable in site_configuration and use that here because different lms sites will have different studio sites.